### DISCUSSION TITLE
How can i manage header and footer?

### DISCUSSION THREAD

**Discussion by saileshkush95** [GrapesJS/grapesjs#5092] [Q&A]
I want to render header and footer in every pages so that user can edit header and footer from any pages. so what it the best way to achieve this.

**Comment by pittleCheung** [comment]
hi,brother. have you solved this problem yet, do have any good solution?

**Comment by artf** [comment]
https://github.com/GrapesJS/grapesjs/discussions/4317

**Comment by Imranmazhar323** [comment]
> I want to render header and footer in every pages so that user can edit header and footer from any pages. so what it the best way to achieve this.

I also want this type of global header !!
